tests/functional/rbac/test_permissions_deleting.py

```diff
@@ -0,0 +1,152 @@
+    """
+    Test that user is still have access if user removed from policy but is in group
+    """
+    cluster_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    group = sdk_client_fs.group_create("test_group", user=[{"id": user.id}])
+    policy = create_policy(
+        sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[user], groups=[group]
+    )
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+    policy.update(user=[{"id": sdk_client_fs.user(username="admin").id}])
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+
+
+def test_remove_user_from_group_but_still_in_another_group(user_sdk: ADCMClient, user, prepare_objects, sdk_client_fs):
+    """
+    Test that user is still have access if user removed from group but is in another group
+    """
+    cluster_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    group = sdk_client_fs.group_create("test_group", user=[{"id": user.id}])
+    another_group = sdk_client_fs.group_create("another_group", user=[{"id": user.id}])
+    policy = create_policy(
+        sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[], groups=[group, another_group]
+    )
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+    policy.update(group=[{"id": another_group.id}])
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+
+
+def test_remove_another_object_from_policy(user_sdk: ADCMClient, user, prepare_objects, sdk_client_fs):
+    """
+    Test that user is still have access if object removed from policy but exists high-level object
+    """
+    cluster_via_admin, service_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    service = user_sdk.service(id=service_via_admin.id)
+    policy = create_policy(
+        sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster, service], users=[user], groups=[]
+    )
+    is_allowed(service, BusinessRoles.ViewConfigurations)
+    policy.update(object=[{"id": cluster.id, "type": "cluster"}])
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+
+
+def test_remove_policy_but_exists_same_policy(user_sdk: ADCMClient, user, prepare_objects, sdk_client_fs):
+    """
+    Test that user is still have access if removed policy but we still have policy with the same rights
+    """
+    cluster_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    policy = create_policy(sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[user], groups=[])
+    second_policy = create_policy(
+        sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[user], groups=[]
+    )
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+    policy.update(object=[{"id": cluster.id, "type": "cluster"}])
```

Comment:
What should this step do? It looks like policy already have this cluster in objects

Reply:
remove service from objects and check that access still allowed

Reply:
but you added only cluster object up here, no?

Reply:
cluster rights allow the same permission for services and components too